pkgs/top-level/all-packages.nix

```diff
@@ -16583,7 +16583,9 @@ in
 
   plano-theme = callPackage ../data/themes/plano { };
 
-  plata-theme = callPackage ../data/themes/plata {};
+  plata-theme = callPackage ../data/themes/plata {
+    inherit (mate) marco;
```

Comment:
If `mateSupport` defaults to `true` and depends on both `gtk3` and `marco`,which default to `null`, then probably `gtk3` should be specified here too, together with `marco`.

But instead of that I would not use `null` as their default value to start with. Then no attribute would be needed here, in `callPackage`.

Reply:
As `mate` isn't really a derivation but a package scope, I prefer to keep this as-is to make it easier to override the `marco` dependency if needed.